tests/cases/AbstractMenuItemsContainerTest.php

```diff
@@ -26,7 +24,11 @@ public function testItems(): void
 		});
 
 		$container = $this->createPartialMockAbstractMenuItemsContainer(null, [
-			$menu, $linkGenerator, $translator, $authorizator, $itemFactory,
+			$menu,
+		$linkGenerator,
```

Comment:
This looks like bad indentation

Reply:
I don't know why phpcsfix did that. :open_mouth: